scripts/cms-any/ci.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+$find = <<<EOT
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    name: CI
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+EOT;
+
+$replace = <<<EOT
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    name: CI
+    # Do not run if this is a pull-request from same repo i.e. not a fork repo
+    # Only check this condition if the account starts with 'silverstripe'
+    if: github.event_name != 'pull_request' || !startsWith(github.event.pull_request.head.repo.owner.login, 'silverstripe') || github.event.pull_request.head.repo.full_name != github.repository
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+EOT;
+
+// Check if a ci.yml file exists, and it exactly matches the $find string then replace it with the $replace string
+// Historically, ci.yml files have not been automatically added by module-standardiser
+// This allows for custom ci.yml files to be added when needed
+$ciPath = '.github/workflows/ci.yml';
+if (check_file_exists($ciPath)) {
+    if (trim(read_file($ciPath)) === trim($find)) {
+        write_file_even_if_exists($ciPath, $replace);
+    }
```

Comment:
This looks like a one-off that won't do anything after this specific run. I don't think we want to commit this as a script that lives in standardiser forever.

I will review this from a "checking that this does what we want to do" lens, but not merge the PR even after I've concluded it's what we want to do.

Reply:
I think there's still some value in having it, simply because of the comment in there that ci.yml has been historically manually added and why that is.  It took me a while to work out why there was no ci.php file, it we be nice to not have the confusion in the future

Reply:
If the intention is to avoid creating it manually, perhaps we should have a separate script that looks for repos that are missing it, but should have it (i.e. they have unit tests, behat tests, linting config, jest tests, etc) and add it if it's missing.

I don't think _this script as written_ is suitable as a script we keep.

Reply:
Hmm, inappropriate solution it turns out - framework doesn't match :-) https://github.com/silverstripe/silverstripe-framework/blob/5/.github/workflows/ci.yml#L12C10-L13C58

Reply:
This is what the list I recommended is for